### DISCUSSION TITLE
Werden Sicherheitslücken gefixt?

### DISCUSSION THREAD

**Discussion by BlackLotus** [Freetz-NG/freetz-ng#956] [Allgemein]
Habe nichts konkretes dazu finden können, aber werden Sicherheitslücken von Geräten die EOL sind aktiv durch freetz gefixt oder werden nur die neusten images/sources genommen und mit den zusätzlichen Features versehen?

**Comment by fda77** [comment]
Nein, Freetz ist nur eine Erweiterung von AVM-FOS. Die Ergüsse von AVM wie zb multid oder ctlmgr die öfter mal Sicherheitslücken (zb via https) haben sind ein Geschäftsgeheimnis und closed source (nicht dass die Chinesen den DHCP Server kopieren!!1).
 Für den "tr064cgi-bug" "Vorfall" der bislang noch keine öffentliche Beschreibung von AVM hat wird aber eine Warnung angezeigt:
![image](https://github.com/Freetz-NG/freetz-ng/assets/471850/55dc2217-aaff-420f-a830-3c9e6f0498c9)
Siehe auch https://github.com/Freetz-NG/freetz-ng/discussions/846 und https://avm.de/service/sicherheitsinfos-zu-updates/  "Details werden zu einem späteren Zeitpunkt veröffentlicht."

Da niemand den Source von dem ganzen Zeug das AVM so programmiert kennt könnten dort auch in aktuellen FOS Versionen noch viele Bugs drin sein - man weiss es nicht.
Alternativ könntest du für alte Hardware Openwrt verwenden, falls unterstützt. Das ist für meine uralte 7320 der einzige Weg Bugfixes und aktuelle Features (multi-ssid, vlan, etc)  zu bekommen
